utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -556,246 +849,265 @@ int main(int argc, char **argv)
 
         for (int i = 0; i < ioBufferSize; i++)
         {
-            *inputi8Temp = (Rpp8s) (((Rpp32s) *inputTemp) - 128);
-            *inputi8_secondTemp = (Rpp8s) (((Rpp32s) *input_secondTemp) - 128);
+            *inputi8Temp = (Rpp8s)(((Rpp32s)*inputTemp) - 128);
+            *inputi8_secondTemp = (Rpp8s)(((Rpp32s)*input_secondTemp) - 128);
             inputTemp++;
             inputi8Temp++;
             input_secondTemp++;
             inputi8_secondTemp++;
         }
     }
 
-    // Run case-wise RPP API and measure time
+    // Run case-wise RPP API and measure time for Unit test
 
     rppHandle_t handle;
     rppCreateWithBatchSize(&handle, noOfImages);
-    clock_t start, end;
-    double start_omp, end_omp;
-    double cpu_time_used, omp_time_used;
+
+    double max_time_used = 0, min_time_used = 500, avg_time_used = 0;
 
     string test_case_name;
 
-    switch (test_case)
+    int count_loop;
+    if (test_type == 0)
     {
-    case 0:
+        count_loop = 1;
+    }
+    else
     {
-        test_case_name = "brightness";
+        count_loop = num_iterations;
+    }
 
-        Rpp32f alpha[images];
-        Rpp32f beta[images];
-        for (i = 0; i < images; i++)
-        {
-            alpha[i] = 1.75;
-            beta[i] = 50;
-        }
+    // case-wise RPP API and measure time script for Unit and Performance test
 
-        // Uncomment to run test case with an xywhROI override
-        /*for (i = 0; i < images; i++)
+    printf("\nRunning %s %d times (each time with a batch size of %d images) and computing mean statistics...", func.c_str(), num_iterations, noOfImages);
+    for (int perfRunCount = 0; perfRunCount < count_loop; perfRunCount++)
+    {
+      
+        switch (test_case)
         {
-            roiTensorPtrSrc[i].xywhROI.xy.x = 0;
-            roiTensorPtrSrc[i].xywhROI.xy.y = 0;
-            dstImgSizes[i].width = roiTensorPtrSrc[i].xywhROI.roiWidth = 100;
-            dstImgSizes[i].height = roiTensorPtrSrc[i].xywhROI.roiHeight = 180;
-        }*/
-
-        // Uncomment to run test case with an ltrbROI override
-        /*for (i = 0; i < images; i++)
+        case 0:
```

Comment:
The case statements are not indented correctly with  the switch case. Please update